omod/src/test/resources/Keycloak/oauth2.properties

```diff
@@ -5,6 +5,8 @@
 #
 clientId=openmrs
 clientSecret=433b9e20-549e-42e7-a05a-6f83dfffa8ba
+publicKey=test
```

Comment:
I actually like this file to look like the real deal, can we put a value that looks like a public key?